examples/framesjs-starter/app/examples/new-api-multi-protocol/frames/route.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-key */
+import { createFrames, Button } from "frames.js/next";
+import { openframes } from "frames.js/middleware";
+import { defaultMiddleware } from "frames.js/core";
+import { getXmtpFrameMessage, isXmtpFrameActionPayload } from "frames.js/xmtp";
+
+const totalPages = 5;
+
+export const frames = createFrames({
+  basePath: "/examples/new-api/frames",
+  initialState: {
+    pageIndex: 0,
+  },
+  middleware: [
+    ...defaultMiddleware,
```

Comment:
is default middleware necessary here?

Reply:
the idea is to extend default functionality - do you think we should explicitly redefine the default middleware here?

Reply:
this is always applied internally so there is no need to pass it here again.